utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -203,6 +203,17 @@ def _remove_mnt_dir(self, mnt_dir):
         if self.mnt_dir is None:
             os.rmdir(mnt_dir)
 
+    def _find_chroot_path(self, mnt_dir):
+        '''
+        Remember actual mounted fs in 'rootfs' for devicemapper
+        '''
+        rootfs_path = os.path.join(mnt_dir, 'rootfs')
+        if os.path.exists(rootfs_path):
+            chroot = rootfs_path
+        else:
+            chroot = mnt_dir
+        return chroot
```

Comment:
you can write all this in just one line: `return rootfs_path if os.path.exists(rootfs_path) else mnt_dir`